app/js/index.js

```diff
@@ -12,6 +11,15 @@ var body;
 var topology;
 var carto;
 var geometries;
+var proj;
+var col = 1;
+var whichMap = 0;
+/* Different integers will correspond to different maps, which accept different .csv data.
+* The default integer, 0, tells the map to take in data for and display the USA,
```

Comment:
this is a good move, just saw it's a bit messy to do enums since they aren't native in JS